.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,109 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there! :wave: :smiley:
+        Thanks for taking the time to fill out this bug report!
+
+        > To expedite issue processing please search open and closed issues before submitting a new one.
+  - type: dropdown
+    id: package-concerned
+    attributes:
+      label: I am opening an issue for
+      options:
+        - Styleguide documentation
+        - Figma
+        - Icons
+        - Styles (CSS, SCSS, ...)
+        - Tokens
+        - Tools (PostCSS, Stylelint, ...)
+    validations:
+      required: true
+  - type: input
+    id: package-version
+    attributes:
+      label: Package version
+      description: Which version of the package is affected?
+      placeholder: ex. @mozaic-ds/styles@1.41.0
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: A clear and concise description of what the bug is
```

Comment:
~~A clear and concise description of what the bug is~~
A clear and concise description of what the bug is.

Reply:
Fixed